common/src/View/ViewPreferencePane.cpp

```diff
@@ -37,6 +37,7 @@
 #include <wx/slider.h>
 #include <wx/stattext.h>
 #include <wx/layout.h>
+#include <wx/valnum.h>
```

Comment:
This should also be added to Prefix.h (so it's included in the precompiled header)